API_ENDPOINTS.md

```diff
@@ -0,0 +1,994 @@
+          "group": 2
+        },
+        "humanState": "Connected",
+        "location": {
+          "ip": "195.201.139.252",
+          "country_code": "DE",
+          "country_name": "Germany",
+          "region_code": "",
+          "region_name": "",
+          "city": "",
+          "zip_code": "",
+          "time_zone": "",
+          "latitude": 51.2993,
+          "longitude": 9.491,
+          "metro_code": 0,
+          "hostname": "node01.betanet.lisk.prolina.org"
+        }
+      }
+    ],
+    "disconnected": [
+      {
+        "ip": "220.178.235.84",
+        "port": 5001,
+        "state": 1,
+        "os": "linux3.2.0-4-amd64",
+        "version": "1.0.0-beta.2",
+        "broadhash": "56b850d6913d2151d97faa5cc255d741e9b7d83430a20c348a3e892c173aa07e",
+        "height": 29902,
+        "osBrand": {
+          "name": "linux",
+          "group": 2
+        },
+        "humanState": "Disconnected",
+        "location": {
+          "ip": "220.178.235.84",
+          "country_code": "CN",
+          "country_name": "China",
+          "region_code": "AH",
+          "region_name": "Anhui",
+          "city": "Hefei",
+          "zip_code": "",
+          "time_zone": "Asia/Shanghai",
+          "latitude": 31.8639,
+          "longitude": 117.2808,
+          "metro_code": 0,
+          "hostname": "220.178.235.84.unknown"
+        }
+      }
+    ]
+  }
+}
+```
+
+## Transactions
+
+### /api/getTransaction
+
+Retrieves transaction details by id.
+
+#### Params
+
+ - `transactionId` - transaction id
+
+#### Sample request
+
+`/api/getTransaction?transactionId=3634383815892709956`
+
+
+#### Sample response
+
+```
+{
+  "success": true,
+  "transaction": {
+    "amount": "0",
+    "asset": {
+      "delegate": {
+        "username": "genesis_51",
+        "publicKey": "01389197bbaf1afb0acd47bbfeabb34aca80fb372a8f694a1c0716b3398db746",
+        "address": "2581762640681118072L"
+      }
+    },
+    "blockId": "6524861224470851795",
+    "confirmations": 17313,
+    "fee": "0",
+    "height": 1,
+    "id": "3634383815892709956",
+    "knownRecipient": null,
+    "knownSender": {
+      "owner": "genesis_51"
+    },
+    "multisignatures": [],
+    "recipientId": "",
+    "recipientPublicKey": "",
+    "senderId": "2581762640681118072L",
+    "senderPublicKey": "01389197bbaf1afb0acd47bbfeabb34aca80fb372a8f694a1c0716b3398db746",
+    "signature": "86e6eed7c8adcdfd0b58d4a718847a8bf4a8c61035003871e8b89d1071123ecacb00a34fd228d9a81074c95265281d578ccb5d72a0f679f7a8066bcae92d090e",
+    "signatures": [
+      "86e6eed7c8adcdfd0b58d4a718847a8bf4a8c61035003871e8b89d1071123ecacb00a34fd228d9a81074c95265281d578ccb5d72a0f679f7a8066bcae92d090e"
+    ],
+    "timestamp": 0,
+    "type": 2
+  }
+}
+```
+
+### /api/getUnconfirmedTransactions
+
+Retrieves list of unconfirmed transactions.
+
+#### Sample request
+
+`/api/getUnconfirmedTransactions`
+
+
+#### Sample response
+
+```
+{
+  "success": true,
+  "transactions": []
```

Comment:
You assume the members of the array to carry the same structure of a single transaction in the response of `/api/getTransaction?transactionId=3634383815892709956` endpoint. but it's not clear. please drop a line of comment to enlighten this assumption. same for all other endpoints in this section